src/map/map.cpp

```diff
@@ -666,6 +666,8 @@ void display_data::set_unit_arrows(std::vector<std::vector<glm::vec2>> const& ar
 
 GLuint load_dds_texture(simple_fs::directory const& dir, native_string_view file_name) {
 	auto file = simple_fs::open_file(dir, file_name);
+	if(!bool(file))
```

Comment:
Are you sure this won't result in the game repeatedly trying to open the file? Doing that operation, even if it fails, can be slow

Reply:
sorry, my bad, time to use peek file then